siliconcompiler/schema/schema_cfg.py

```diff
@@ -2988,6 +2991,53 @@ def schema_option(cfg):
     return cfg
 
 
+############################################
+# Dependency information
+############################################
+def schema_dependency(cfg):
+
+    dependency = 'default'
+
+    scparam(cfg, ['dependency', dependency, 'path'],
+            sctype='str',
+            scope='global',
+            shorthelp="Dependency source path",
+            switch="-dependency_path 'dependency <str>'",
+            example=[
+                "cli: -dependency_path "
+                "'freepdk45_data ssh://git@github.com/siliconcompiler/freepdk45/'",
+                "api: chip.set('dependency', "
+                "'freepdk45_data', 'path', 'ssh://git@github.com/siliconcompiler/freepdk45/')"],
+            schelp="""
+            Dependency source path
+            """)
+
+    scparam(cfg, ['dependency', dependency, 'commitid'],
+            sctype='str',
+            scope='global',
+            shorthelp="Dependency source commitid",
+            switch="-dependency_commitid 'dependency <str>'",
+            example=[
+                "cli: -dependency_commitid 'freepdk45_data 07ec4aa'",
+                "api: chip.set('dependency', 'freepdk45_data', 'commitid', '07ec4aa')"],
+            schelp="""
+            Dependency source commitid""")
+
+    scparam(cfg, ['dependency', dependency, 'name'],
+            sctype='str',
+            scope='global',
+            shorthelp="Dependency name",
+            switch="-dependency_name 'dependency <str>'",
+            example=[
+                "cli: -dependency_name 'freepdk45_data freepdk45'",
+                "api: chip.set('dependency', 'freepdk45_data', 'name', 'freepdk45')"],
+            schelp="""
+            Dependency name in cache: name-commitid
+            Should be unique if no commitid specified""")
```

Comment:
is this needed if the `dependency` is also a name?

Reply:
Just using the name of the dependency would also work of course.
Again might be confusing with the python package name.
For example so far I was using `siliconcompiler_data` as the python package name for the metadata but `siliconcompiler` as the name for the actual content.

Reply:
Then nevermind I can just change it.